src/doubleTrouble.js

```diff
@@ -24,7 +24,16 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(array, target) {
-  // write code here
+  let counter = 0;
```

Comment:
is counter required for this solution? can you implement solution without the counter?